tests/requirements.txt

```diff
@@ -1,5 +1,3 @@
-black
-isort
```

Comment:
isort and black should also be removed from `run_tests.cmd` and `run_tests.sh`.

I'd be fine with having a separate script to run them, but automatically including them in all test runs seems wrong to me.

Reply:
I made a different script in #301 I can port it over or leave this PR w/o one for now and leave that discussion for that PR.